src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,13 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
```

Comment:
The method 'write' should be declared in a separate 'service' package following the project structure guidelines.